pkg/virt-launcher/virtwrap/converter/network.go

```diff
@@ -47,12 +47,22 @@ func CreateDomainInterfaces(vmi *v1.VirtualMachineInstance, domain *api.Domain,
 	networks := indexNetworksByName(nonAbsentNets)
 
 	for i, iface := range nonAbsentIfaces {
-		net, isExist := networks[iface.Name]
+		_, isExist := networks[iface.Name]
 		if !isExist {
 			return nil, fmt.Errorf("failed to find network %s", iface.Name)
 		}
 
-		if iface.Binding != nil || iface.SRIOV != nil || iface.Slirp != nil {
+		var tapDomainAttachment bool
```

Comment:
+1 for moving it to helper, maybe it can be reused in podNIC code as well.
Why would it be necessary to call it twice?

Since it checks that the binding type is non tap, I think we can simplify it as follows:
```
ifaceAttachmentType := c.DomainAttachmentByInterfaceName[iface.Name]
tapDomainAttachment := ifaceAttachmentType == v1.Tap
```

When `DomainAttachmentByInterfaceName` is nil or interface name not exist, the returned key value will be the default value, in this case empty string.